### ISSUE TITLE
`inc` not set in navigation.

### ISSUE THREAD

**Issue by lucasb-eyer** [strands-project/strands_navigation#159]
@Jailander 

There are multiple possible paths to reach [this line](https://github.com/strands-project/strands_navigation/blob/hydro-devel/topological_navigation/scripts/navigation.py#L258) which _do not_ set the `inc` variable, and thus the line fails with an exception:

```
[ERROR] [WallTime: 1416824559.134127] Exception in your execute callback: local variable 'inc' referenced before
 assignment
Traceback (most recent call last):
  File "/opt/ros/hydro/lib/python2.7/dist-packages/actionlib/simple_action_server.py", line 299, in executeLoop
    self.execute_callback(goal)
  File "/opt/ros/hydro/lib/topological_navigation/navigation.py", line 161, in executeCallback
    self.navigate(goal.target)
  File "/opt/ros/hydro/lib/topological_navigation/navigation.py", line 258, in navigate
    tries+=inc
UnboundLocalError: local variable 'inc' referenced before assignment
```

**Comment by hawesie** [comment]
Ditto!

```
[ERROR] [WallTime: 1416825204.021972] Exception in your execute callback: local variable 'inc' referenced before assignment                                                        │························
Traceback (most recent call last):                                                                                                                                                 │························
  File "/opt/ros/hydro/lib/python2.7/dist-packages/actionlib/simple_action_server.py", line 299, in executeLoop                                                                    │························
    self.execute_callback(goal)                                                                                                                                                    │························
  File "/opt/ros/hydro/lib/topological_navigation/navigation.py", line 161, in executeCallback                                                                                     │························
    self.navigate(goal.target)                                                                                                                                                     │························
  File "/opt/ros/hydro/lib/topological_navigation/navigation.py", line 258, in navigate                                                                                            │························
    tries+=inc                                                                                                                                                                     │························
UnboundLocalError: local variable 'inc' referenced before assignment                                                                                                               │························

```

**Comment by lucasb-eyer** [comment]
As a workaround to get Karl started, I just set it to 1 right before its use, made Karl go to sleep (charging), stopped everything, undone my code change and restarted everything. Seems to work so far.

**Comment by Jailander** [comment]
pushing a fix right now

**Comment by Jailander** [comment]
#162 I can't believe that happened today this was introduced on Thursday and never happened sorry should be fixed now

**Comment by Jailander** [comment]
#162 fixed this issue, Issue #161 remains open, so I will close this
